server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/DockerAwsS3.java

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.blob.objectstorage;
+
+import java.net.URI;
+
+public class DockerAwsS3 {
+    private final URI scalityEndpoint;
+
+    public DockerAwsS3(URI scalityEndpoint) {
+        this.scalityEndpoint = scalityEndpoint;
+    }
+
+    public URI scalityEndpoint() {
+        return scalityEndpoint;
```

Comment:
is it AWS S3 or scality ?

Reply:
scality is an implementation of AWS S3, kind of, I do not see the issue here

Reply:
our implementation has nothing specific to scality so we should stick to s3 naming